### ISSUE TITLE
Pass UID with messages

### ISSUE THREAD

**Issue by vu0tran** [thusfresh/switchboard#50]
I know I can get the message_id but how about the UID of the email?

**Comment by vu0tran** [comment]
Got it. In the messageId property, it's just merged, e.g. "messageId"=>"INBOX!612907036?47339" then "47339" is the UID.

**Comment by jtmoulia** [comment]
Great that you found a solution, but a warning: that value is supposed to be opaque and might change in future versions. If so, we'll add an explicit way to access the UID.

**Comment by vu0tran** [comment]
Yeah, I figured. It doesn't get passed on "messages" which is what I needed, but I found a way to a hack at it. Thanks.

I feel like UID is pretty vital though. Without the UID, you can't defer any work to client side imap clients.
